add_npm_package.sh

```diff
@@ -7,7 +7,14 @@ TITO_TAG=foreman-nightly-nonscl-rhel7
 DISTRO=${TITO_TAG##*-}
 BASE_DIR=${4:-foreman}
 
-PACKAGE_NAME=nodejs-${NPM_MODULE_NAME##*/}
+PACKAGE_MODULE=${NPM_MODULE_NAME##*/}
+PACKAGE_VENDOR=${NPM_MODULE_NAME%%/*}
+PACKAGE_VENDOR=${PACKAGE_VENDOR##@}
+if [[ $NPM_MODULE_NAME == @* ]]; then
```

Comment:
Should this check if `PACKAGE_VENDOR` is non-empty?

Reply:
It never is -- it will be equal the package module name for a non-namespaced package, but I thought that checking vendor == module is also wrong, as someone might have `@foo/foo`

Reply:
Pehaps it should check if `$PACKAGE_NAME == */*` because we really do this because of the slash.

Reply:
updated.